cmd/kubelet/app/server.go

```diff
@@ -264,6 +264,7 @@ func Run(s *options.KubeletServer, kcfg *KubeletConfig) error {
 	if kcfg == nil {
 		cfg, err := UnsecuredKubeletConfig(s)
 		if err != nil {
+			glog.Errorf("Unable to generate a kubelet config: %v", err)
```

Comment:
The caller prints this to stderr: https://github.com/kubernetes/kubernetes/blob/791d160b42302a7ae54b2def5f05b4583a63b8fa/cmd/kubelet/kubelet.go#L48

This seems to be the standard for all cmd/* applications.